favorites.py

```diff
@@ -103,12 +103,24 @@ class FavFileMgr(object):
     """Handle file actions."""
 
     @classmethod
+    def fav_file(cls, filename, data=False, read=False, write=False):
+        """Read or write currently handled favorite list."""
+
+        if read:
+            with open(filename) as file:
+                # Allow C style comments and be forgiving of trailing commas
+                content = sanitize_json(file.read(), True)
+            return json.loads(content)
+
+        elif write:
+            with open(filename, "w") as file:
+                json.dump(data, file, sort_keys=True, indent=4, separators=(',', ': '))
+
+    @classmethod
     def check_aliases(cls, favs):

```

Comment:
I didn't realize at the time, but this function should not be alias specific.  It should be called something like `upgrade_format` ans we may use it to upgrade to other formats in the future.

Reply:
Ok upgrade_version?